components/MW/metgene_metabolite_table/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import { z } from 'zod'
+import { valHooks } from 'jquery'
+
+// A unique name for your data type is used here
+export const MetgeneMetaboliteTable = MetaNode.createData('MetgeneMetaboliteTable')
+  // Human readble descriptors about this node should go here
+  .meta({
+    label: 'Metgene metabolite table',
+    description: 'Metgene metabolite table, rendered',
+  })
+  // this should have a codec which can encode or decode the data type represented by this node
+  //  using zod, a compile-time and runtime type-safe codec can be constructed
+  .codec(z.any())
```

Comment:
Please add the runtime type of this, z.any() is not suitable for production

Reply:
Would the runtime type be MetgeneMetaboliteTable in this case?